Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/06-aks-supporting.md

```diff
@@ -4,6 +4,7 @@ The following will be created:
 * Azure Container Registry (supporting-infra.tf)
 * Azure Key Vault (supporting-infra.tf)
 * Private Link Endpoints for ACR and Key Vault
+* Public DNS Zone (supporting-infra.tf), this is optional and required only if you have custom hostnames assigned to endpoints and worklods deployed in AKS
```

Comment:
Add that this is used only in the Blue/green scenario

Reply:
fixed